src/client/clientSchema.graphql

```diff
@@ -1,18 +1,17 @@
 extend interface NewMeeting {
   localPhase: NewMeetingPhase!
   localStage: NewMeetingStage!
-  sidebarCollapsed: Boolean
 }
 
 extend type RetrospectiveMeeting {
   localPhase: NewMeetingPhase!
   localStage: NewMeetingStage!
-  sidebarCollapsed: Boolean
   reflectionsInFlight: [RetroReflection]
 }
 
 extend type Team {
   contentFilter: String
+  sidebarCollapsed: Boolean
```

Comment:
`sidebarCollapsed` is only used in the new meeting context at the moment. Should I rename this to `meetingSidebarCollapsed`?

Reply:
yeah, `isMeetingSidebarCollapsed` gets my vote. i'm always a fan of long but clear var names, and booleans that start with a question